languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34110,125 @@ though, and has since been slightly modified)-->
+              </antipattern>
+              <pattern>
+                  <token postag="SENT_START"/>
+                  <marker>
+                      <token regexp="yes">moi|toi</token>
+                  </marker>
+                  <token postag="[RAVNJPZCD].*" postag_regexp="yes"/>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\2,</suggestion>
+              <example correction="Moi,"><marker>Moi</marker> je ne le sais pas.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token postag="SENT_START"/>
+                  <marker>
+                      <token regexp="yes">lui|elle|eux|nous|vous|elles</token>
+                  </marker>
+                  <token postag="R.* pers.* suj.*" postag_regexp="yes"/>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\2,</suggestion>
+              <example correction="Lui,"><marker>Lui</marker> il ne le sais pas.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token postag="V.*" postag_regexp="yes"/>
+                  <token postag="R.* pers.* suj.*" postag_regexp="yes"/>
+                  <token>que</token>
+                  <token postag="R.*" postag_regexp="yes"/>
+                  <token postag="V.*" postag_regexp="yes" skip="-1"/>
+              </antipattern>
+              <antipattern>
+                  <token>qu'</token>
+                  <token>elle</token>
+                  <token>que</token>
+              </antipattern>
+              <pattern>
+                  <marker>
+                      <token postag="R.* pers.* suj.*" postag_regexp="yes"/>
+                      <token>que</token>
+                  </marker>
+                  <token postag="R.*" postag_regexp="yes"/>
+                  <token postag="V.*" postag_regexp="yes" skip="-1"/>
+                  <token postag="V.*" postag_regexp="yes"/>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\1, \2</suggestion>
+              <example correction="Elle, que"><marker>Elle que</marker> je connaissais bien, n'est plus là aujourd'hui.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token postag="SENT_START"/>
+                  <marker>
+                      <token postag="A.*" postag_regexp="yes"><exception regexp="yes">comment|quand|pourquoi</exception></token>
+                      <token>il</token>
+                  </marker>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\2, \3</suggestion>
+              <example correction="Dehors, il"><marker>Dehors il</marker> fait beau.</example>
+          </rule>
```

Comment:
Please take a closer look at the rule editor results. This subrule is promising, too, but there are still significant FPs:

> Tantôt il court, tantôt il marche

Reply:
All the necessary changes have been done.

Reply:
I corrected the punctuation that should be excuded.